striprtf/striprtf.py

```diff
@@ -190,17 +190,25 @@ def rtf_to_text(text, encoding="utf-8", errors="strict"):
                     c = int(arg)
                     if c < 0:
                         c += 0x10000
-                    out = out + chr(c).encode(encoding, errors)
+                    try:
+                        chr1 = chr(c).encode(encoding, errors).decode(encoding).encode('utf-8', errors)
+                    # because some flavors of rtf seem to allow utf8 inside of cp1252
+                    except UnicodeEncodeError as e:
+                        chr1 = chr(c).encode('utf-8', errors)
```

Comment:
I think you should just execute this line unconditionally.

What is the point of first trying to encode it through `encoding`?  It's a Unicode character — just encode it directly into UTF-8.

Reply:
better yet, skip the encoding step entirely as commented below